### ISSUE TITLE
SearchPanel  not look good in SimpleTheme

### ISSUE THREAD

**Issue by ZSYMAX** [AvaloniaUI/AvaloniaEdit#311]
![8c29d2892369da547baf8f4098751d5](https://user-images.githubusercontent.com/38749563/222642059-73725689-31bc-4e68-af80-79e9b3bd3c9f.png)

**Comment by danipen** [comment]
Yeah, AvaloniaEdit is supposed to work with the fluent theme by default.

I think I can make an improvement making the buttons in the search panel theme-agnostic. I'll take a look.

**Comment by danipen** [comment]
@ZSYMAX can you try #312? It should work fine now. I created themes for simple and fluent. You just need to change[ this line ](https://github.com/AvaloniaUI/AvaloniaEdit/blob/a54f1db5237b981661ac6a8ae1c957eb389e3b8f/src/AvaloniaEdit/AvaloniaEdit.xaml#L6)in order to switch to simple theme for AvaloniaEdit, and use:
```xaml
<MergeResourceInclude Source="/Themes/Simple/Base.xaml" />
```

**Comment by ZSYMAX** [comment]
![image](https://user-images.githubusercontent.com/38749563/222699026-47140654-2a04-4bee-bc66-859be92236b8.png)

I have tried, but there is a small problem. The foreground color of textbox does not follow the system theme change.

**Comment by danipen** [comment]
I'll take a look. What is the mechanism you use to change the theme? Can you post here the code?

**Comment by danipen** [comment]
Well, I reviewed the implementation and I'm not doing anything special with textboxes and labels in the SearchPanel. They are just default textboxes and labels, so they should behave as other controls in the UI. 

@Gillibald any ideas those controls are not reacting to theme changes? (Sorry I don't know the Simple/Fluent theme handling since in our production app we're using our owner theming mechanism).

Is working fine for the fluent theme, or also failing?

**Comment by ZSYMAX** [comment]
![image](https://user-images.githubusercontent.com/38749563/223000677-78f8ab51-94b6-44a6-b9b4-04d87171142f.png)

this is my code in simple thime.
it is work good in fluent theme. @danipen

**Comment by danipen** [comment]
It's working ok for me.
![image](https://user-images.githubusercontent.com/501613/223039985-bed7a190-2ddd-496e-a638-174563c533d1.png)

Note that you don't need to include the `avares://AvaloniaEdit/Themes/Simple/Base.xaml` in the demo's `App.xaml`. You just need to change which ResourceDictionary you want to use in here: https://github.com/AvaloniaUI/AvaloniaEdit/blob/a54f1db5237b981661ac6a8ae1c957eb389e3b8f/src/AvaloniaEdit/AvaloniaEdit.xaml#L6

**Comment by danipen** [comment]
In any case, I'm going to bring all the needed AvaloniaEdit styles to the Demo app, so maybe it's more clear to use that way.

**Comment by danipen** [comment]
Ok, done. Now all you need to do is using `<StyleInclude Source="avares://AvaloniaEdit/Themes/Simple/AvaloniaEdit.xaml" />`
![image](https://user-images.githubusercontent.com/501613/223046061-89dbf178-9990-4e42-a93c-998acc8a8535.png)
